variables.tf

```diff
@@ -41,4 +48,40 @@ variable "create_namespace" {
   type        = bool
   default     = false
   description = "When true, creates the namespace for the Terraform Cloud Agent."
-}
\ No newline at end of file
+}
+
+variable "limits_cpu" {
+  type        = string
+  default     = "1"
+  description = "CPU hard limits."
+}
+
+variable "limits_memory" {
+  type        = string
+  default     = "2Gi"
```

Comment:
I'm worried that this is too low.  I'm going to see if I can find sensible requests and limits.